valkey.conf

```diff
@@ -300,6 +300,51 @@ tcp-keepalive 300
 #
 # tls-session-cache-timeout 60
 
+################################### RDMA ######################################
+
+# By default, RDMA is disabled. To enable it, the "rdma-port" configuration
+# directive can be used to define RDMA-listening ports.
+#
+# rdma-port 6379
+# rdma-bind 192.168.1.100
+
+# The RDMA receive transfer buffer is 1M by default. It can be set between 64K and 16M.
+# Note that page size aligned size is preferred.
+#
+# rdma-rx-size 1048576
+
+# The RDMA completion queue will use the completion vector to signal completion events
+# via hardware interrupts. A large number of hardware interrupts can affect CPU performance.
+# It is possible to tune the performance using rdma-comp-vector.
+#
+# Example 1. a) Pin hardware interrupt vectors [0, 3] to CPU [0, 3].
+#            b) Set CPU affinity for valkey to CPU [4, X].
+#            c) Any valkey server uses a random RDMA completion vector.
+# All valkey servers will not affect each other and will be isolated from kernel interrupts.
+#
+#   SYS    SYS    SYS    SYS  VALKEY VALKEY     VALKEY
+#    |      |      |      |      |      |          |
+#  CPU0   CPU1   CPU2   CPU3   CPU4   CPU5   ... CPUX
+#    |      |      |      |
+#  INTR0  INTR1  INTR2  INTR3
+#
+# Example 2. a) Pin hardware interrupt vectors [0, X] to CPU [0, X].
+#            b) Set CPU affinity for valkey to CPU [0, X].
+#            c) Valkey server [M] uses RDMA completion vector [M].
+# A single CPU handles hardware interrupts, the RDMA completion queue, and the valkey server.
+# This avoids overhead and function calls across multiple CPUs.
+#
+# VALKEY VALKEY VALKEY VALKEY VALKEY VALKEY     VALKEY
+#    |      |      |      |      |      |          |
+#  CPU0   CPU1   CPU2   CPU3   CPU4   CPU5  ...  CPUX
+#    |      |      |      |      |      |          |
+#  INTR0  INTR1  INTR2  INTR3  INTR4  INTR5      INTRX
+#
+# Use 0 and positive numbers to specify the RDMA completion vector, or specify -1 to allow
+# the server to use a random vector for a new connection. The default vector is -1.
+#
+# rdma-comp-vector 0
```

Comment:
`comp-vector` is same as [`struct ibv_cq *ibv_create_cq(struct ibv_context *context, int cqe, void *cq_context,  struct ibv_comp_channel *channel,  int comp_vector);`](https://man7.org/linux/man-pages/man3/ibv_create_cq.3.html), this may be friendly to guys  who are familiar with RDMA programming. This is the reason I use `rdma-comp-vector`.

From valkey user side, which one do you prefer? Should I link this man page here?

Reply:
```
The CQ will use the completion vector comp_vector for signaling completion events
```

Leads me to believe we should use completion-vector, since it seems like the human friendly name. We don't normally abbreviate config names. comp_vector seems like the prototype is just trying to compress it (which might make sense in the context of rdma developers since they will write it a lot).